### PR TITLE
fix

### PR DESCRIPTION
**Pull request by dragonylffly**

### CODE REVIEW

**Comment by dragonylffly** [comment]
@PFZheng

File: src/braft/fsm_caller.cpp [hunk 1]
```diff
@@ -78,10 +78,13 @@ int FSMCaller::run(void* meta, bthread::TaskIterator<ApplyTask>& iter) {
                 max_committed_index = -1;
                 g_commit_tasks_batch_counter << counter;
                 counter = 0;
-                batch_size = FLAGS_raft_fsm_caller_commit_batch;
```
Comment (PFZheng) [inline src/braft/fsm_caller.cpp:81]: This line is used to determine if the batch size is changed manually.

Comment (dragonylffly) [reply src/braft/fsm_caller.cpp:81]: It seems that FLAGS_raft_fsm_caller_commit_batch not dynamically revised by using "FLAGS_raft_fsm_caller_commit_batch=" or "SetCommandLineOption" in the codes, and batch_size has got its value in line 67

Comment (PFZheng) [reply src/braft/fsm_caller.cpp:81]: gflags registered with BRPC_VALIDATE_GFLAG can be dynamically changed.

**Comment by dragonylffly** [comment]
@PFZheng updated

**Pull request merged**